src/data/apiClient.js

```diff
@@ -1,5 +1,7 @@
 import { getAuthenticatedAPIClient } from '@edx/frontend-auth';
 
+import NewRelicService from './services/NewRelicService';
+
```

Comment:
We need to use the `NewRelicLoggingService` from `@edx/frontend-logging` like we are doing in https://github.com/edx/frontend-app-learner-portal/pull/13/files#diff-75f76d91899c96b633e6282624e91fbd